### PR TITLE
L1 sdk support

### PR DESCRIPTION
**Pull request by brikendr**

### CODE REVIEW

File: huddlyproto/proto/huddly.proto [hunk 1]
```diff
@@ -0,0 +1,231 @@
+syntax = "proto3";
```
Comment (pkhansen) [inline huddlyproto/proto/huddly.proto:1]: Some stuff missing in regards to what Geir was working on, but it can also be included in my PR :)

**Review by pkhansen — APPROVED** [review]
Great!

File: huddlyproto/proto/huddly.proto (hunk 1 above)
Comment (brikendr) [reply huddlyproto/proto/huddly.proto:1]: We will probably have to rely on one source of truth about the proto file. It is most likely that the proto files will not live here but on falcon-interface instead.

Comment (pkhansen) [reply huddlyproto/proto/huddly.proto:1]: Ah ok, was thinking that we might keep it here until the falcon-interface stuff is figured out.

**Pull request closed without merging**